src/markdown/ndoch_2016.md

```diff
@@ -0,0 +1,64 @@
+# National Day of Civic Hacking 2016 Resources
+
+> Welcome to Code for Denver's 2016 National Day of Civic Hacking. This page aims to provide you with resources to help you have a successful and enjoyable event.
+
+## Agenda
+|              |  
+| ------------ | ---------
+| 8:00         |  		Breakfast
+| 8:15         |  		Welcome
+| 8:30         | 		  Education on the issue and challenge spaces
+| 9:00         | 		  Q & A
+| 9:30         | 		  Hacking: Break into groups according to challenges
+| 12:00        | 		  Lunch and check in
+| 12:45        | 		  More hacking
+| 4:00         | 		  Report out
+| 4:30         | 		  Next steps toward implementation
+| 5:00         | 		  Adjourn
+| 5:00 - 10:00 |      After event hack-a-thon. More hacking for those who just can't stop!!!!
+
+___
+
+## Links
+
+**Slide Decks**
+
+- [Intro](https://docs.google.com/presentation/d/1pzOxyGq3yunT4cq7aESPD-X39545lq3SdqfRRuk7KP4)
+- [Hackshop](https://drive.google.com/open?id=1TxiN2OGW47Tr3Q6s6L6uJbVk0jwpo72ae_94YyMTD0E)
+- [Closing](https://drive.google.com/open?id=1DhwsBiq9eQFQ36-HyZliEzaAMur7DCNUKL1e6ne0sbI)
+
+**2016 NdoCH challenge**
+
+- [Apply for Affordable Housing](https://www.codeforamerica.org/events/national-day-2016/challenge-applying-for-subsidized-housing)
+
+**Non-programming hacking**
```

Comment:
The below challenges all can be parts of the Applying for Affordable Housing challenge above. Do we want to signify that at all?